modules/nw-infw-operator-rules-object.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-node-firewall-operator.adoc
+
+:_content-type: CONCEPT
+[id="nw-ingress-node-firewall-operator-rules-object_{context}"]
+= Ingress node firewall rules object
+
+The fields for the Ingress Node Firewall Operator rules object are described in the following table:
+
+.Ingress node firewall rules object
+[cols=".^2,.^2,.^6a",options="header"]
+|====
+|Field|Type|Description
+
+|`metadata.name`
+|`string`
+|The name of the CR object. The name of the config object must be `ingressnodefirewall`.
+
+|`interfaces`
+|`array`
+|The fields for this object specify the interfaces to apply the firewall rules to. For example `- en0`.
+`- en1`
+
+|`nodeSelector.matchLabels`
+|`array`
+|You can use `nodeSelector` `matchLabels` to select the nodes to apply the firewall rules to. Set the value of your named `matchLabels` to `true` to apply the rule.
+|====
+
+[discrete]
+[id="nw-infw-ingress-rules-object_{context}"]
+=== Ingress object configuration
+
+The values for the `ingress` object are defined in the following table:
+
+.`ingress` object
+[cols=".^3,.^2,.^5a",options="header"]
+|====
+|Field|Type|Description
+
+|`sourceCIDRs`
+|`array`
+|You can configure multiple CIDRs from different address families.
+
+[NOTE]
+====
+Different CIDRs allow you to use the same order rule. In the case that there are multiple `IngressNodeFirewall` objects for the same nodes and interfaces with overlapping CIDRs, the `order` field will specify which rule is applied first.
+====
+
+|`rules`
+|`array`
+|Ingress firewall `rules.order` are ordered starting at `1` for each `source.CIDR` with up to 100 rules per CIDR. Lower order rules are executed first.
+
+`protocolConfig.protocol` supports the following protocols: TCP, UDP, SCTP, ICMP and ICMPv6.
+
+[NOTE]
+====
+For `protocol.ICMP`, ICMP and ICMPv6 rules can match against ICMP and ICMPv6 type and code. For `TCP`, `UDP`, and `SCTP` rules can match against single DstPort or a range of ports using `<start-end>` format.
```

Comment:
The actual range would be (start : end-1) so if you define "30000-32767" say, range in effect would be 30000-32766